.github/workflows/checks.yaml

```diff
@@ -92,13 +99,13 @@ jobs:
           echo "RUSTC_WRAPPER=sccache" >> $GITHUB_ENV
 
       - name: Build the rbuilder
-        run: cargo build
+        run: cargo build --features="${{ matrix.features }}"
```

Comment:
Are we doubling or CI times with this? Do we need to CI test that the logs are correct?

Reply:
No, they run as seperate jobs in parallel. Yes ci should check all features.